py-rattler/tests/unit/test_solver.py

```diff
@@ -0,0 +1,33 @@
+# type: ignore
+import os.path
+
+import pytest
+from rattler import (
+    solve,
+    Channel,
+    ChannelConfig,
+    MatchSpec,
+    RepoDataRecord,
+    SparseRepoData,
+)
+
+
+@pytest.mark.asyncio
+async def test_solve():
+    linux64_chan = Channel("conda-forge", ChannelConfig(f"http://localhost:{8813}/"))
```

Comment:
I think using localhost here can be confusing because you are not actually connecting to it right?
```suggestion
    linux64_chan = Channel("https://conda.anaconda.org/conda-forge", ChannelConfig())
```

Reply:
Or is it possible to not specify the `ChannelConfig`? That would be even nicer.

Reply:
Currently you have to define a `ChannelConfig` but it does default to conda-forge, but I would also like to make `ChannelConfig` optional, actually I'm going to do it right now.